migrations-1/marketplace1.sql

```diff
@@ -0,0 +1,192 @@
+
+-- Run marketplace2.sql in marketplace2 container first , then run this file
+-- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+CREATE EXTENSION postgres_fdw;
+CREATE SERVER node1 FOREIGN DATA WRAPPER postgres_fdw OPTIONS (host 'marketplace2', dbname 'ds_db');
```

Comment:
K8s doesn't name StatefulSet pods in this manner. It's `$(statefulset name)-$(ordinal)` you should agree with Mariam in the `$(statefulset name)`.